lib/features/give/pages/organization_list_page.dart

```diff
@@ -0,0 +1,264 @@
+      case CollecGroupType.church:
+        title = locals.church;
+        break;
+      case CollecGroupType.charities:
+        title = locals.charity;
+        break;
+      case CollecGroupType.campaign:
+        title = locals.campaign;
+        break;
+      case CollecGroupType.artists:
+        title = locals.artist;
+        break;
+      default:
+        break;
+    }
+
+    return title;
+  }
+
+  Expanded _buildGivingButton({
+    required String title,
+    VoidCallback? onPressed,
+  }) {
+    return Expanded(
+      flex: 0,
+      child: Padding(
+        padding: const EdgeInsets.all(20),
+        child: ElevatedButton(
+          onPressed: onPressed,
+          style: ElevatedButton.styleFrom(
+            disabledBackgroundColor: Colors.grey,
+          ),
+          child: Text(title),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildListTile({
+    required VoidCallback onTap,
+    required bool isSelected,
+    required String title,
+    required CollecGroupType type,
+  }) =>
+      ListTile(
+        key: UniqueKey(),
+        onTap: onTap,
+        selected: isSelected,
+        selectedTileColor: getHighlightColor(type),
+        leading: Icon(
+          getIconByType(type),
+          color: AppTheme.givtBlue,
+        ),
+        title: Text(title),
+      );
+
+  Widget _buildFilterType(OrganisationBloc bloc, AppLocalizations locals) =>
+      Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: [
+          FilterSuggestionCard(
+            isFocused: bloc.state.selectedType == CollecGroupType.church.value,
+            title: locals.church,
+            icon: 'assets/images/church.png',
+            activeIcon: 'assets/images/church_focus.png',
+            color: AppTheme.givtLightBlue,
+            onTap: () => bloc.add(
+              OrganisationTypeChanged(
+                CollecGroupType.church.value,
+              ),
+            ),
+          ),
+          FilterSuggestionCard(
+            isFocused:
+                bloc.state.selectedType == CollecGroupType.charities.value,
+            title: locals.charity,
+            icon: 'assets/images/charity.png',
+            activeIcon: 'assets/images/charity_focus.png',
+            color: AppTheme.givtYellow,
+            onTap: () => bloc.add(
+              OrganisationTypeChanged(
+                CollecGroupType.charities.value,
+              ),
+            ),
+          ),
+          FilterSuggestionCard(
+            isFocused:
+                bloc.state.selectedType == CollecGroupType.campaign.value,
+            title: locals.campaign,
+            icon: 'assets/images/campaign.png',
+            activeIcon: 'assets/images/campaign_focus.png',
+            color: AppTheme.givtOrange,
+            onTap: () => bloc.add(
+              OrganisationTypeChanged(
+                CollecGroupType.campaign.value,
+              ),
+            ),
+          ),
+          FilterSuggestionCard(
```

Comment:
I don't think that Artists is visible right now as a filter

Reply:
#18 has this change